substratevm/src/com.oracle.svm.core/src/com/oracle/svm/core/jfr/JfrEmergencyDumpFeature.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright (c) 2025, 2025, Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 2025, 2025, Red Hat Inc. All rights reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.  Oracle designates this
+ * particular file as subject to the "Classpath" exception as provided
+ * by Oracle in the LICENSE file that accompanied this code.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+package com.oracle.svm.core.jfr;
+
+import com.oracle.svm.core.VMInspectionOptions;
+import com.oracle.svm.core.feature.AutomaticallyRegisteredFeature;
+import com.oracle.svm.core.feature.InternalFeature;
+
+import jdk.graal.compiler.api.replacements.Fold;
+
+import org.graalvm.nativeimage.ImageSingletons;
+
+/**
+ * The JFR emergency dump mechanism uses platform-specific implementations (see
+ * {@link JfrEmergencyDumpSupport}).
+ */
+@AutomaticallyRegisteredFeature
+public class JfrEmergencyDumpFeature implements InternalFeature {
+    @Fold
+    static boolean isPresent() {
```

Comment:
unused

Reply:
It gets used in SubstrateJVM.java to guard things related to the emergency dump [[1](https://github.com/oracle/graal/pull/11530/files#diff-978c849c061c0a70780cd4568e8ee9a024fdb40c5b1211d7a15bad00fc9387bfR756),[2](https://github.com/oracle/graal/pull/11530/files#diff-978c849c061c0a70780cd4568e8ee9a024fdb40c5b1211d7a15bad00fc9387bfR599)].